packages/malloy-db-bigquery/src/bigquery_connection.ts

```diff
@@ -699,6 +704,7 @@ export class BigQueryConnection
       location: this.location,
       maximumBytesBilled:
         this.config.maximumBytesBilled || MAXIMUM_BYTES_BILLED,
+      jobTimeoutMs: Number(this.config.timeoutMs) || TIMEOUT_MS,
```

Comment:
Technically this parameter is a string, probably to avoid integer overflow, but since we using a number before I guess that also works, but if we find that people want really big timeouts, then we may want to change that.